src/main/java/com/odysseusinc/arachne/datanode/exception/PermissionDeniedException.java

```diff
@@ -22,7 +22,7 @@
 
 package com.odysseusinc.arachne.datanode.exception;
 
-public class PermissionDeniedException extends Exception {
+public class PermissionDeniedException extends RuntimeException {
```

Comment:
There are lot of usage of this. Why this changes required?

Reply:
On one side - Security is checked/handled by the spring so it is overkill to force user handle it too..
and it is inconvenient to use methods with throws non-runtime exceptions in the lamdas.
So I think it will be good to mark them as Runtime to avoid necessary try {
}catch
in business logic.